pure-pso/templates/plugin/node-server.yaml

```diff
@@ -43,6 +43,15 @@ spec:
       # https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
       dnsPolicy: ClusterFirstWithHostNet
       hostPID: true
+{{- if eq .Values.app.prereq true}}
```

Comment:
I believe we have to add the variable to cockroach-operator too, so that it will know to add the init container to the DB pods.

Reply:
Never mind, just realized it's using the same exact variable names so it's fine.